optimum/intel/openvino/configuration.py

```diff
@@ -537,6 +554,68 @@ def get_default_quantization_config(
     return None
 
 
+def _merge_ignored_scopes(
+    ignored_scope_1: Union[Dict[str, List[str]], None], ignored_scope_2: Union[Dict[str, List[str]], None]
+) -> Dict[str, List[str]]:
+    """
+    Merges two ignored scopes represented as dictionaries. If a key exists in both dictionaries, the corresponding lists
+    are merged and duplicates are removed.
+    Args:
+        ignored_scope_1 (`dict` or `None`):
+            The first ignored scope dictionary.
+        ignored_scope_2 (`dict` or `None`):
+            The second ignored scope dictionary.
+    Returns:
+        Merged ignored scope dictionary.
+    """
+    if ignored_scope_1 is None:
+        return copy.deepcopy(ignored_scope_2) if ignored_scope_2 is not None else None
+    if ignored_scope_2 is None:
+        return copy.deepcopy(ignored_scope_1)
+    merged_ignored_scope = {}
+    for key in set(ignored_scope_1) | set(ignored_scope_2):
+        merged_ignored_scope[key] = list(set(ignored_scope_1.get(key, []) + ignored_scope_2.get(key, [])))
+    return merged_ignored_scope
+
+
+def _apply_default_ignored_scope_config(
+    model_id_or_path: str, quantization_config: "OVPipelineQuantizationConfig"
+) -> "OVPipelineQuantizationConfig":
+    """
+    Applies default ignored scope configuration to the given quantization configuration based on the model ID or path.
+    Args:
+        model_id_or_path (`str`):
+            id of the model or path to it.
+        quantization_config (`OVPipelineQuantizationConfig`):
+            The quantization configuration to which the default ignored scope will be applied.
+    Returns:
+        Updated quantization configuration with the default ignored scope applied.
+    """
+    quantization_config_copy = None
+    model_id_candidates = _get_model_id_candidates(model_id_or_path)
+    for model_id in model_id_candidates:
+        default_ignored_scopes_per_model = _DEFAULT_IGNORED_SCOPE_CONFIGS.get(model_id)
+        if not default_ignored_scopes_per_model:
+            continue
+        quantization_config_copy = quantization_config.clone()
+        for ov_model_name, default_ignored_scope in default_ignored_scopes_per_model.items():
+            q_config = quantization_config_copy.quantization_configs.get(
+                ov_model_name, quantization_config_copy.default_config
```

Comment:
we could add a check to make sure `quantization_config` is in instance of `OVPipelineQuantizationConfig` (to avoid any potential issue here)